### PR TITLE
Specify region in prometheus.  Addresses #2436.

### PR DESCRIPTION
**Pull request by DailyDreaming**
Addresses #2436.

### CODE REVIEW

**Comment by joelarmstrong** [comment]
Thanks a ton! I should have a chance to test it within the next day or so. I'll let you know if it works or not!

**Comment by DailyDreaming** [comment]
@joelarmstrong Awesome, thank you!

**Comment by DailyDreaming** [comment]
@joelarmstrong I know you're busy with workflow runs atm, but if you get a chance this week to review this, I'd appreciate it!

I implemented what I was thinking of.  If it doesn't make sense to you, please let me know and I'll try the sentinel file idea (the status API used natively makes a sentinel file though and that's what it checks for the existence of indirectly, though I could try to do it more directly if that's what you were suggesting)!

File: src/toil/utils/toilStatus.py [hunk 1]
```diff
@@ -184,21 +188,29 @@ def getStatus(jobStoreName):
...
         try:
             with jobstore.readSharedFileStream('succeeded.log') as successful:
-                return 'COMPLETED'
+                pass
+            return 'COMPLETED'
         except NoSuchFileException:
             try:
-                with jobstore.readSharedFileStream('failed.log') as failedLog:
-                    return 'ERROR'
+                with jobstore.readSharedFileStream('failed.log') as failed:
```
Comment (joelarmstrong) [inline src/toil/utils/toilStatus.py]: Sorry, I know this isn't relevant to this PR, but I just noticed: is `failed.log` ever actually deleted on resume? I can't see anywhere that the deletion happens. If not, a failed but successfully restarted workflow will forever get the FAILED status (until it completes).

Comment (DailyDreaming) [reply src/toil/utils/toilStatus.py]: That's a really good point.  I'll create an issue and a corresponding PR.  Thanks for catching this!

**Pull request merged**